src/components/atoms/HeartAnimationButton/heart-button.styled.ts

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+export const StyledButton = styled.div`
```

Comment:
Nice "styledButton" that is actually a div :D

Reply:
Oh good catch!